verifier/testreport.py

```diff
@@ -699,6 +708,38 @@ def characterize_results_by_options(self, test_list, category):
 
         return results
 
+
+    def characterize_plural_rules_tests(self, test_list, results):
+        # look for consistencies with plural rules test
+        for test in test_list:
+            label = test['label']
+            sample = test['input_data']['sample']
+            sample_type = 'integer sample'
+            if sample.find('c') >= 0:
+                sample_type = 'compact sample'
+            elif sample.find('.') >= 0:
+                sample_type = 'float sample'
+            elif sample.find('.e') >= 0:
+                sample_type = 'exponential sample'
```

Comment:
For the string argument to the `.find()` function, does it represent a substring, or does it represent a regex pattern?

Reply:
Not a regex.